package.json

```diff
@@ -11,7 +11,7 @@
     "bugs": {
         "url": "https://github.com/Becklyn/mojave/issues"
     },
-    "version": "5.0.0",
+    "version": "5.1.0",
```

Comment:
Please don't change that, as it will automatically updated when releasing a new version.